docs/image/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM ubuntu-debootstrap:14.04
+
+RUN apt-get update && apt-get install --yes python python-pip postgresql-9.3 make postgresql-server-dev-9.3  python-dev libffi-dev git  && apt-get clean
+
+RUN easy_install --upgrade pip
```

Comment:
Our standard one-liner for installing `pip` is this:
```Dockerfile
RUN curl -sSL https://raw.githubusercontent.com/pypa/pip/1.5.6/contrib/get-pip.py | python -
```
Could you change this line to match that?